applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_FIC_element.cpp

```diff
@@ -152,6 +153,9 @@ void UPwSmallStrainFICElement<TDim, TNumNodes>::InitializeNonLinearIteration(con
         Variables.B                  = b_matrices[GPoint];
         Variables.F                  = deformation_gradients[GPoint];
         Variables.ConstitutiveMatrix = constitutive_matrices[GPoint];
+        ConstitutiveLawUtilities::SetConstitutiveParameters(
```

Comment:
This function call is no longer necessary, since the constitutive matrices are already calculated (they are only retrieved here from the vector). I think there will be only a few places left now after the changes of the PR for the lists of constitutive matrices. Basically everywhere we retrieve the Constitutive Matrix from a list (i.e. `Variables.ConstitutiveMatrix = constitutive_matrices[GPoint];`) we don't need this call anymore. I think it will be there in ~3 places for diff order and ~3 places for non-diff order.

Reply:
Thank you. Removed in many places. But not all calls. ;)